antarest/core/config.py

```diff
@@ -184,6 +185,10 @@ def from_dict(cls, data: JSON) -> "StorageConfig":
             auto_archive_dry_run=data.get("auto_archive_dry_run", defaults.auto_archive_dry_run),
             auto_archive_sleeping_time=data.get("auto_archive_sleeping_time", defaults.auto_archive_sleeping_time),
             auto_archive_max_parallel=data.get("auto_archive_max_parallel", defaults.auto_archive_max_parallel),
+            snapshot_retention_days=data.get(
+                "snapshot_variant_lifespan_days",
```

Comment:
This name is not the one you chose so plz change this

Reply:
Actually I see that we don't have unit tests for configuration parsing, maybe we could add some ?